Server/reportpreview.R

```diff
@@ -41,7 +43,8 @@ output$gen_info <- renderText({
     "</h4>",
     "<h4><b>Published:</b>",
     pkg_GenInfo$published,
-    "</h4>"
+    "</h4>",
+    "<h4> Using riskmetric version",riskinfo$loadedversion,"</h4>"
```

Comment:
Hi @Robert-Krajcik, it would be best to move this information below as until that point the information displayed is about the package being reviewed.

How about below the overall risk?
https://github.com/pharmaR/risk_assessment/blob/f8e79988d58d8e65db0bbc98676451bcf1bc9a15/Server/reportpreview.R#L57

We can put a label in bold: `**riskmetric version:** [some_version]`.

Reply:
This PR fixes the problem =) I have only one comment above.